src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingoverride/InputMissingOverrideBadOverrideFromObject.java

```diff
@@ -44,6 +44,25 @@ public int hashCode() {
      * {@inheritDoc}
      */
     public int hashCode(); // violation 'include @java.lang.Override annotation when '@inheritDoc''
+
+    /**
+     * {@inheritDoc}
+     */
+    @Deprecated // violation 'include @java.lang.Override annotation when '@inheritDoc''
+    public String toString();
+
+    @SuppressWarnings("")
+    /**
+     * {@inheritDoc}
+     */
+    @Deprecated
+    public boolean equals(Object a); // ok, because it is dangling javadoc
+
+    @SuppressWarnings("")
+    /**
+     * {@inheritDoc}
+     */
+    public HashEq2 clone(Object a); // ok, because it is dangling javadoc
```

Comment:
> dangling

I am not sure what it mean, please rephrase.

Reply:
this is term from idea inspection - https://stackoverflow.com/questions/43375177/warning-dangling-javadoc-comment
basically that means misplaced javadoc

Reply:
It's basically our `InvalidJavadocPosition` check.

````
$ cat TestClass.java
public class TestClass {
    @SuppressWarnings("")
    /**
     * {@inheritDoc}
     */
    public HashEq2 clone(Object a);
}

$ cat TestConfig.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
<module name="InvalidJavadocPosition"/>
    </module>
</module>

$ java -jar checkstyle-9.2.1-all.jar -c TestConfig.xml TestClass.java
Starting audit...
[ERROR] TestClass.java:3:5: Javadoc comment is placed in the wrong location. [InvalidJavadocPosition]
Audit done.
Checkstyle ends with 1 errors.
````

Reply:
renamed